apps/ledger-live-mobile/src/components/Nft/NftList.tsx

```diff
@@ -64,6 +64,7 @@ export function NftList({ data }: Props) {
             : 1
         }
         mr={(index + 1) % NB_COLUMNS > 0 ? 6 : 0}
+        testID={"wallet_nft_gallery_list_item"}
```

Comment:
Just a small thing - we're using the convention for naming test IDs with `-` instead of `_`. Could you change this and the others to use those?

Reply:
fixed